src/index.html

```diff
@@ -5,8 +5,368 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap"  rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+      <header class="catalog__header header">
+        <a href="#home">
+          <img
+            class="header__logo"
+            src="images/logo.png"
+            alt="moyo"
+           >
+        </a>
+        <nav>
+          <main class="header__nav nav">
+            <section class="nav__list is-active" href="#">Apple</section>
+            <section class="nav__list"  href="#">Samsung</section>
+            <section class="nav__list"  href="#">Smartphones</section>
+            <section
+               class="nav__list nav-hover"
+               href="#"
+               data-qa="nav-hover"
+            >Laptops & Computers</section>
+            <section class="nav__list"  href="#">Gadgets</section>
+            <section class="nav__list"  href="#">Tablets</section>
+            <section class="nav__list"  href="#">Photo</section>
+            <section class="nav__list"  href="#">Video</section>
+          </main>
```

Comment:
Menu items isn't main and sections, you need to use tag ul and li instead of main and section here